i18n/utils.py

```diff
@@ -1,9 +1,11 @@
-from babel import Locale, UnknownLocaleError
+from babel import Locale
+from babel import UnknownLocaleError
```

Comment:
We usually group imports from the same package into a single import statement. I'd like to continue with that style if we can.

Reply:
We currently have both `pre-commit` and `make` file linting and testing on the repo. This is causing formatting conflicts. @dpoirier

Reply:
https://github.com/asottile/reorder_python_imports

Reply:
Removed `pre-commit` from the repository. `Makefile` is now the source of truth.

Reply:
done. re-ordered imports